invoke-ai/InvokeAI
# feat(ui, nodes): add fields for CLIPEmbedModel and FluxVAEModel

## Summary

* update flux model node to take inputs for CLIP Embed and VAE - this removes the brittle way in which they were hard-coded, which will hopefully make it easier for users to upload and use models that were not directly downloaded from our starter model list
* update default FLUX workflow to have these fields. this will improve validation in the front-end when a user is missing a model; they will not be able to Invoke and hopefully more easily figure out why.
* I had to re-download my Flux VAE while working on this but am not sure if it was related to my changes. The error was around the config_path so I may have had an out of date install in my DB.

## Related Issues / Discussions



## QA Instructions

Restart server (make sure that DB has this default workflow loaded) and run workflow. Should be required to make a selection from your installed models for all four models for the Flux model node.

## Merge Plan



## Checklist
